packages/htmlbars-runtime/lib/morph.js

```diff
@@ -21,6 +21,7 @@ function HTMLBarsMorph(domHelper, contextualElement) {
   this.childNodes = null;
```

Comment:
this.state = {}

we should confirm that the pre-alloc is actually use by most morphs, otherwise it should likely be lazy.

It may also want to be an emptyObject